test/common/upstream/load_stats_reporter_test.cc

```diff
@@ -145,7 +146,7 @@ TEST_F(LoadStatsReporterTest, ExistingClusters) {
     envoy::config::endpoint::v3::ClusterStats foo_cluster_stats;
     foo_cluster_stats.set_cluster_name("foo");
     foo_cluster_stats.set_cluster_service_name("bar");
-    foo_cluster_stats.set_total_dropped_requests(5);
+    foo_cluster_stats.set_total_dropped_requests(7);
```

Comment:
The test changes that update the dropped-requests/drop-overload stats are required because previously the cluster info wasn't stateful and essentially was reset after each invocation of the stats.
I think that the use of "add" methods to the stats is better reflected at the moment.